src/services/git.js

```diff
@@ -0,0 +1,40 @@
+const path = require('path')
+const ProcessManager = require('../utils/processManager')
+const {validateCommand} = require('../utils/folders')
+const gitRegex = '^git\\s[^&|;]*$'
+
+class GitService {
+
+  constructor () {
+    this.processManager = new ProcessManager()
+    this.websocket = null
+    this.currentSharedFolder = null
+  }
+
+  setWebSocket (websocket) {
+    this.websocket = websocket
+  }
+
+  sharedFolder (currentSharedFolder, readOnly) {
+    this.currentSharedFolder = currentSharedFolder
+    this.readOnly = readOnly
+  }
+
+  /**
+   * Create and check if command is valid
+   * @param action
+   * @param id
+   * @param key
+   * @param name
+   * @param payload
+   * @param callback
+   */
+  command (args, callback) {
+    const options = {cwd: this.currentSharedFolder, shell: true}
```

Comment:
I removed the `validateCommand` from here cause I didn't really understand it,
`args.script` is the full command, that would I guess need to be parsed for getting it through a filter.
Could you readd `validateCommand` in this PR?